test/test_utils/ec2.py

```diff
@@ -774,11 +775,14 @@ def execute_ec2_training_test(
     if "neuron" in ecr_uri:
         connection.run(f"sudo modprobe -r neuron  && sudo modprobe -i neuron")
 
-    return connection.run(
+    LOGGER.info(f"execute_ec2_training_test running {ecr_uri}, with cmd {test_cmd}")
```

Comment:
Was this logging call intentionally moved? This looks like it still belongs in front of the `connection.run` method call on L#728.

Reply:
Yes it signifies the execution of the test command but not the start of the container. I mistakenly added before the running of the container last time.